documentation-website/Writerside/topics/Get-started-with-Exposed-DAO.md

```diff
@@ -0,0 +1,280 @@
+# Get started with Exposed's DAO API
+
+<show-structure for="chapter,procedure" depth="2"/>
+<tldr>
+    <var name="example_name" value="get-started-with-exposed-dao"/>
+    <include from="lib.topic" element-id="code_example"/>
+</tldr>
+
+<web-summary>
+    In this tutorial, you’ll learn how to create and query a table in Kotlin with Exposed's DAO API.
+</web-summary>
+<link-summary>
+    Learn how to create and query tables in Kotlin with Exposed's DAO API.
+</link-summary>
+
+In this tutorial, you’ll learn how to use Exposed’s Data Access Object (DAO) API to store and retrieve data in a
+relational database by building a simple console application.
+
+By the end of this tutorial, you’ll be able to do the following:
+
+- Configure a database connection using an in-memory database.
+- Define database tables and corresponding DAO entities.
+- Perform basic CRUD (Create, Read, Update, and Delete) operations using object-oriented style.
+
+<include from="Get-Started-with-Exposed.topic" element-id="prerequisites"/>
+<var name="project_name" value="exposed-dao-kotlin-app"/>
+<include from="Get-Started-with-Exposed.topic" element-id="create-new-project"/>
+
+## Add dependencies
+
+
+<procedure>
+Before you start using Exposed, you need to provide dependencies to your project.
+<step>
+
+Navigate to the **gradle/libs.versions.toml** file and define the Exposed and H2 version and libraries:
+
+```kotlin
+[versions]
+//...
+exposed = "%exposed_version%"
+h2 = "%h2_db_version%"
+
+[libraries]
+//...
+exposed-core = { module = "org.jetbrains.exposed:exposed-core", version.ref = "exposed" }
+exposed-dao = { module = "org.jetbrains.exposed:exposed-dao", version.ref = "exposed" }
+exposed-jdbc = { module = "org.jetbrains.exposed:exposed-jdbc", version.ref = "exposed" }
+h2 = { module = "com.h2database:h2", version.ref = "h2" }
+```
+
+- The `exposed-core` module provides the foundational components and abstractions needed to work with databases in a
+type-safe manner and includes the DSL API.
+- The `exposed-dao` module allows you to work with the Data Access Object (DAO) API.
+- The `exposed-jdbc` module is an extension of the <code>exposed-core</code> module that adds support for Java 
+Database Connectivity (JDBC).
+
+</step>
+<step>
+
+Navigate to the **app/build.gradle.kts** file and add the Exposed and H2 database modules into the `dependencies` block:
+
+```kotlin
+dependencies {
+    //...
+    implementation(libs.exposed.core)
+    implementation(libs.exposed.dao)
+    implementation(libs.exposed.jdbc)
+    implementation(libs.h2)
+    //...
+}
+```
+</step>
+<step>
+    <include from="lib.topic" element-id="intellij_idea_load_gradle_changes"/>
+</step>
+</procedure>
+
+## Configure a database connection
+
+Every database access using Exposed is started by obtaining a connection and creating a transaction.
```

Comment:
I wouldn't say "access is started", maybe something like "Every time you access a database using Exposed, you start by..."